docs/indepth/installation.rst

```diff
@@ -403,3 +403,35 @@ You have also ensured that the ``flocker-deploy`` command line tool is able to c
 Next you may want to perform the steps in :doc:`the tutorial <./tutorial/moving-applications>` , to ensure that your nodes are correctly configured.
 Replace the IP addresses in the ``deployment.yaml`` files with the IP address of your own nodes.
 Keep in mind that the tutorial was designed with local virtual machines in mind, and results in an insecure environment.
+
+
+Experiment on a remote testing cluster
+--------------------------------------
+
+Requires - e.g. Credentials for cloud providers of your choice.
+Maybe permissions you need?
+
+You can quickly start up a cluster to test on.
+Use the flocker-provision tool (see installing flocker-cli).
+
+This installs CentOS 7 on Rackspace nodes.
+
+Options are:
+* Cloud provider? Maybe
+   * For each cloud provider, credential options (e.g. rackspace secret key, region)
+* Number of nodes
+* Instance details - have defaults which work but user can decide instance size etc. - follow-up?
+
+For example:
+
+flocker-provision create --driver rackspace --rackspace-secret=secret --num-agent-nodes=3 --rackspace-region=us-west-1
+> Creating... 90%
```

Comment:
Will this progress report be possible? (If not) What helpful output could we give to show that something is going on?